.github/workflows/main.yml

```diff
@@ -9,15 +9,15 @@ jobs:
     name: 🧪 Test
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2-beta
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: '10'
+          node-version: 16
```

Comment:
This may not work out of the box. I'm unsure why this was using node 10, but it might have to do with matching the version to the one that the cloud functions run on.

Reply:
Looks like it succeeded 🥳. Though there were several warnings unrelated talking about packages being deprecated/out of date.